.github/actions/maven-build-and-tag/action.yml

```diff
@@ -72,27 +72,44 @@ runs:
         ${{ inputs.verbose }} && NTP=''
         echo "result=--show-version --settings settings.xml $NTP" >> $GITHUB_OUTPUT
 
+    - name: Set is_preview env variable
+      id: set-is-preview-env-variable
+      env:
+        IS_PREVIEW: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'Create preview') }}
```

Comment:
I wonder if "preview" could not be enough as a label (as it's already a verb), and actually, maybe this should be extracted as an input so that it can integrate better with the caller repository pull request labels, wdyt?

Reply:
@atchertchian I added the input and changed the default label to `preview`. Thanks for the comment!